examples/tutorials/advanced/Privacy Attacks/Tutorial 1 - Black box model inversion.ipynb

```diff
@@ -0,0 +1,714 @@
+      "text/plain": [
+       "<Figure size 720x360 with 12 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "attack(attacker, target_model, test_data, range(6))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "You can see that we have successfully recreated the input images\n",
+    "without having access to any training data.\n",
+    "While handwritten digits are not the most private of data,\n",
+    "this attack would have worked equally well on a model to classify faces\n",
+    "or MRI results.\n",
```

Comment:
Maybe this is less assertive but as suggestive: "this attack could also be applied on much more sensitive data including face images or MRI records."